src/index.html

```diff
@@ -1,18 +1,82 @@
 <!doctype html>
 <html lang="en">
   <head>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/main.scss"
```

Comment:
SCSS files should not be linked directly in HTML. Instead, compile them to CSS and link the CSS file.